internals/models_test.py

```diff
@@ -449,22 +552,22 @@ def test_get_approvals(self):
     self.assertEqual(1, len(actual))
     self.assertEqual(models.Approval.REVIEW_REQUESTED, actual[0].state)
 
-    actual = models.Approval.get_approvals(
-        order=-models.Approval.set_on, limit=10)
-    self.assertEqual(2, len(actual))
-    self.assertEqual(models.Approval.APPROVED, actual[0].state)
-    self.assertEqual(models.Approval.REVIEW_REQUESTED, actual[1].state)
-
-    actual = models.Approval.get_approvals(
-        order=-models.Approval.set_on, limit=1)
-    self.assertEqual(1, len(actual))
-    self.assertEqual(models.Approval.APPROVED, actual[0].state)
-
-    actual = models.Approval.get_approvals(
-        order=-models.Approval.set_on, states=[models.Approval.APPROVED],
-        limit=10)
-    self.assertEqual(1, len(actual))
-    self.assertEqual(models.Approval.APPROVED, actual[0].state)
+    # actual = models.Approval.get_approvals(
+    #     order=-models.Approval.set_on, limit=10)
+    # self.assertEqual(2, len(actual))
+    # self.assertEqual(models.Approval.APPROVED, actual[0].state)
+    # self.assertEqual(models.Approval.REVIEW_REQUESTED, actual[1].state)
+
+    # actual = models.Approval.get_approvals(
+    #     order=-models.Approval.set_on, limit=1)
+    # self.assertEqual(1, len(actual))
+    # self.assertEqual(models.Approval.APPROVED, actual[0].state)
+
+    # actual = models.Approval.get_approvals(
+    #     order=-models.Approval.set_on, states=[models.Approval.APPROVED],
+    #     limit=10)
+    # self.assertEqual(1, len(actual))
+    # self.assertEqual(models.Approval.APPROVED, actual[0].state)
```

Comment:
Is this worth keeping rather than refactoring or deleting?

Reply:
Gah, I forgot to delete these.  The sorting functionality for approvals moved into `sorted_by_pending_request_date()` and `sorted_by_review_date()`, which are tested on lines  169 to 181 of this file.